committeeoversightapp/views.py

```diff
@@ -82,7 +84,21 @@ class EventListJson(BaseDatatableView):
     max_display_length = 500
 
     def filter_queryset(self, qs):
-        # based on example at https://pypi.org/project/django-datatables-view/
+        # in order to filter by categories and committees, grab the detail
+        # object type and its pk from the url and filter
```

Comment:
This is the wonkiest part of this PR, imo: in order to filter hearings by category or committee, we need to pass this `EventListJson` view that context. The way I found to do that was to modify the `order_list_json` url to accept a variable, and then pass both the type of filter and object ID in the template.

The wonky part is that this view is then parsing the URL path into two parts, the `detail_type` and `pk`, in order to filter the table. Curious what you think here @hancush, and if you see a way to do this without URL parsing!

Reply:
ooh! i had to solve a similar problem in pensions. i ended up [adding url parameters to my ajax request, with values from my template context](https://github.com/datamade/bga-pensions/blob/0179475b15baeda43749afeff59967e0d6d45186/templates/index.html#L293), then [using them to filter the queryset in the view](https://github.com/datamade/bga-pensions/blob/0179475b15baeda43749afeff59967e0d6d45186/pensions/views.py#L331). i think it's a little more explicit than parsing the url... but i'm open to your input, too! what do you think?

Reply:
Using url parameters is a much nicer approach! Switched it.